src/main/kotlin/lotto/LottoPrize.kt

```diff
@@ -0,0 +1,16 @@
+package lotto
+
+enum class LottoPrize(val matchCount: Int, val prizeMoney: Int) {
+    FOURTH_PLACE(3, 5_000),
+    THIRD_PLACE(4, 50_000),
+    SECOND_PLACE(5, 1_500_000),
+    FIRST_PLACE(6, 2_000_000_000),
+    NOTHING(0, 0);
+
+    companion object {
+        fun getPrize(matchingNumbers: Int): Int {
+            return values().firstOrNull { it.matchCount == matchingNumbers }?.prizeMoney ?: 0
+        }
```

Comment:
등급을 찾는것과 거기서 값을 꺼내는 기능은 분리해도 좋을 것 같아요.
```suggestion
        fun getPrize(matchingNumbers: Int): Int {
            return findByNumber(matchingNumbers)?.prizeMoney ?: 0
        }
```

Reply:
넵! 말씀해주신대로 변경하겠습니다!